app/controllers/application_controller.rb

```diff
@@ -1,9 +1,13 @@
 class ApplicationController < ActionController::Base
   before_action :current_user
+  before_action :configure_permitted_parameters, if: :devise_controller?
 
   private
 
-  def current_user
-    @current_user = User.first
+  # def current_user
+  #   @current_user = User.first
+  # end
```

Comment:
* Kindly remove all commented code from your codebase as it makes your codebase bulky and unprofessional